src/build/_util.py

```diff
@@ -65,3 +67,18 @@ def check_dependency(
 
 def parse_wheel_filename(filename: str) -> re.Match[str] | None:
     return _WHEEL_FILENAME_REGEX.match(filename)
+
+
+if typing.TYPE_CHECKING:
```

Comment:
```suggestion
if typing.TYPE_CHECKING:  # pragma: no cover
```

Reply:
Shouldn't we just disable coverage for all `if typing.TYPE_CHECKING`?

Reply:
Yeah, we should.